scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/task/SchedulerVars.java

```diff
@@ -49,5 +49,6 @@
     PA_TASK_PROGRESS_FILE,
     PA_TASK_REPLICATION,
     PA_TASK_SUCCESS,
-    PA_USER
+    PA_USER,
+    PA_REST_URL
```

Comment:
PA_SCHEDULER_REST_URL? there are many REST API and thus URL available by default. The Workflow Catalog REST API is an example? Regarding this last, it could be interesting to also pass it to a task. More generally, all REST URL of services deployed by the Scheduler should be passed or a mean to reconstruct it given.